biodatasets/n2c2_2018_track2/n2c2_2018_track2.py

```diff
@@ -0,0 +1,436 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and Ayush Singh (singhay).
```

Comment:
initially the template requested that participants add names here, but then we decided to use github to track contributions. can you make this consistent with the template please https://github.com/bigscience-workshop/biomedical/blob/master/templates/template.py#L2

Reply:
done